### ISSUE TITLE
bug(resource): \n in ReadTextValueAsString

### ISSUE THREAD

**Issue by flavens** [dasch-swiss/0807-mls-app-web#9]
Line feed character is not considered as line break but as actual character part of the text:

E.G:
<img width="718" alt="Screen Shot 2019-04-03 at 11 26 42" src="https://user-images.githubusercontent.com/29532462/55468685-36289d00-5604-11e9-9911-330f450581ba.png">

**Comment by flavens** [comment]
@subotic Sepideh had the same issue: the problem was already in the XML of the data before importing in Knora. Do you have the same?

**Comment by subotic** [comment]
No, the original XML data contains true line feeds. In the bulk-import XML, the line feeds are also there.

**Comment by flavens** [comment]
fix: https://github.com/dhlab-basel/Knora/pull/1296

**Comment by subotic** [comment]
Thanks. I will need to re-import everything when the patch is merged.
